packages/plugins/src/index.ts

```diff
@@ -336,6 +339,9 @@ const modules = [
   coingecko,
   orphanPlatforms,
   zeusnode,
+  convex,
+  fraxlend,
+  velodrome2
```

Comment:
```suggestion
  velodrome2,
```
Let's add a trailing comma so the next addition doesn't bring SCM noise.